### ISSUE TITLE
Handling unescaped quotes

### ISSUE THREAD

**Issue by 0cd** [mholt/PapaParse#358]
I noticed that when copying from Excel or Google Sheets (copy happens as TSV lines), any literal double-quotes in a value are not escaped unless there's also a line-break in the value. 

Parsing the copied data using PapaParse is then problematic if a value begins with a double-quote. PapaParse thinks of the value as being enclosed in double-quotes. If the second double quote is at the end of the value, the double-quotes are lost. If the second double-quote is in the middle of the value or missing, there is a parse error. 

I believe when you paste such data back into Excel or Google Sheet, they handle it by treating double-quotes as part of the value when there's no line-break in the value. PapaParse could potentially do the same?

I don't know what the TSV standard (if there is one) says, but currently it's error-prone to parse data copied from Excel & Google Sheet using PapaParse, when dealing with values that contain a double-quote as first character.

**Comment by spelunk** [comment]
This also shows up on the demo site.  Try putting in

```
1	2	"3"	4
```

Excel only wraps text in double quotes when the particular value contains the field or record separator.  For TSV the record separator is the newline and the field separator is the tab.  Should there be an "Excel" mode that follows excel behavior?

**Comment by datafunk** [comment]
FYI, I had the same issue with CSV and my "offline" solution for this was to ensure that if one field is wrapped in quotes, then all fields must be wrapped in quotes. I know, it'd be nice to have Papa handle this for instances where we have no control over the input, but I thought I throw it in there in case it helps, short of getting down to it and solving and submitting a PR, which right now I am not prepared to do

**Comment by spelunk** [comment]
@datafunk I ended up rolling my own CSV parser.  This behavior does not conform to [RFC 4180](https://tools.ietf.org/html/rfc4180) so the parser would have to break the standard to work with Excel

**Comment by datafunk** [comment]
@spelunk yep, I realise that

**Comment by veryhidden** [comment]
ran into the same problem today :( @spelunk can you share it?

**Comment by rsilvergun** [comment]
I'm not sure if I'm seeing the same/similar issues. I wanted papa parse to keep the quotes in it's parse data when I set quotes to false but it doesn't seem to do that. I can't find any way to preserve quotes in the parsed output. I'm might just be doing something wrong (I've got a stack overflow question out here: http://stackoverflow.com/questions/42817095/preserving-quotes-with-papa-parse to ask).
